### PR TITLE
Dstara/bug fixes

### PR DESCRIPTION
**Pull request by DimitrisStaratzis**
Same as [here](https://github.com/TileDB-Inc/TileDB-Trino/pull/81)

### CODE REVIEW

File: src/test/java/com/facebook/presto/plugin/tiledb/TestTileDBQueries.java [hunk 1]
```diff
@@ -1575,7 +1551,7 @@ private void create1DVectorNoAttribute(String arrayName)
     {
         QueryRunner queryRunner = getQueryRunner();
         String createSql = format("CREATE TABLE %s(" +
-                "x bigint WITH (dimension=true) " +
+                "x integer WITH (dimension=true) " +
```
Comment (ihnorton) [inline src/test/java/com/facebook/presto/plugin/tiledb/TestTileDBQueries.java:1554]: Why the change here? Is this from -Java change?

Comment (DimitrisStaratzis) [reply src/test/java/com/facebook/presto/plugin/tiledb/TestTileDBQueries.java:1554]: Just aligning it with Trino, no particular reason for the change.

**Comment by DimitrisStaratzis** [comment]
> Minor ? but LGTM

Removed "minor"
